src/test/groovy/org/prebid/server/functional/model/request/setuid/SetuidRequest.groovy

```diff
@@ -15,6 +15,8 @@ class SetuidRequest {
     String gdpr
     @JsonProperty("gdpr_consent")
     String gdprConsent
+    String gpp
+    String gppSid
```

Comment:
SID is a finite number AFAIK. Definitely not a string. Probably should be an ENUM.

Reply:
It's comma-separated list of integers.

Reply:
Yeah, but you can serialize that accordingly. If it's a comma-separated list, you are better off with a List of Enums. Or you'll end up with a string, that consists of magic numbers. Which we don't really like to see.